samples/bluetooth/peripheral_ans/README.rst

```diff
@@ -0,0 +1,26 @@
+.. zephyr:code-sample:: ble_peripheral_ans
+   :name: Peripheral ANS
+   :relevant-api: bluetooth
+
+   Send notification using Alert Notification Service (ANS).
+
+Overview
+********
+
+This sample demonstrates the usage of ANS by acting as a peripheral periodically sending
+notifications to the connected remote device.
+
+Requirements
+************
+
+* A board with Bluetooth LE support
+* Smartphone with BLE app (ADI Attach, nRF Connect, etc.) or dedicated BLE sniffer
```

Comment:
```suggestion
* Smartphone with Bluetooth LE app (ADI Attach, nRF Connect, etc.) or dedicated Bluetooth LE sniffer
```

IMO we should avoid using the term "BLE", and instead rely on the official term "Bluetooth LE"